### ISSUE TITLE
Merlin 系统的路由器可以支持这个脚本吗？

### ISSUE THREAD

**Issue by Askwind** [wukongdaily/tvhelper#12]
请问运行梅林系统 网件Rax50 可以运行这个脚本吗？或者后期可以编译一个可以使用在梅林系统上的版本吗？

**Comment by wukongdaily** [comment]
～～～这个嘛，得抽空研究一下子，我对梅林系统还不够熟悉哈，隐约的觉得可以实现。需要研究一下Entware 这个。梅林里好像是可以用这个插件的，然后貌似需要配合u盘。有了它应该就可以使用opkg 来安装adb了，就是不晓得这个Entware 那里头的软件源是否包含adb，一旦有了adb，其他的程序应该就是普通shell脚本了。就应该能兼容梅林了

**Comment by Askwind** [comment]
好的！ 期待悟空的后续的进展

**Comment by Askwind** [comment]
> ～～～这个嘛，得抽空研究一下子，我对梅林系统还不够熟悉哈，隐约的觉得可以实现。需要研究一下Entware 这个。梅林里好像是可以用这个插件的，然后貌似需要配合u盘。有了它应该就可以使用opkg 来安装adb了，就是不晓得这个Entware 那里头的软件源是否包含adb，一旦有了adb，其他的程序应该就是普通shell脚本了。就应该能兼容梅林了

是的 Merlin是可以安装Entware 环境的，而且也有ARM版的 Alist可以使用，只要能再安装个xiaoyaalist清理工具就可以按照你视频中那样使用一个轻量级的xiaoya套装了。
